src/app/util/attribute-metadata.h

```diff
@@ -60,14 +60,14 @@ union EmberAfDefaultAttributeValue
     constexpr EmberAfDefaultAttributeValue(uint16_t val) : defaultValue(val) {}
 
     /**
-     * Points to data if size is more than 2 bytes.
-     * If size is more than 2 bytes, and this value is NULL,
+     * Points to data if size is more than 4 bytes.
+     * If size is more than 4 bytes, and this value is NULL,
      * then the default value is all zeroes.
      */
     const uint8_t * ptrToDefaultValue;
 
     /**
-     * Actual default value if the attribute size is 2 bytes or less.
+     * Actual default value if the attribute size is 4 bytes or less.
```

Comment:
this value should be converted to uint32_t then

Reply:
Ops. I reverted change to this comment. I am keeping this uint16_t since this structure is used for MaxMinDefault values. I am not changing this since there are other things that would need to be accounted for with min/max type stuff